src/components/charts/hubblePlot/index.jsx

```diff
@@ -59,6 +63,7 @@ class HubblePlot extends React.Component {
       mousePosX: null,
       mousePosY: null,
       showTooltip: false,
+      showNudge: false,
```

Comment:
`showNudge` matches `showTooltip` in most cases except that it does not display on hover.